### ISSUE TITLE
Text on top of images - gradient underlay needed

### ISSUE THREAD

**Issue by anevins12** [WordPress/twentynineteen#6]
Once we add text on top of images, we'll need to have a consistent underlay/overlay design pattern that ensures text is accessible on any image.

The minimum contrast that is required for an overlay just under 0.55 opacity of black. I recommend adding a solid opacity of black underneath the text and then adding a gradient above or below that text, dependant on where it is displayed on the image.

Example: http://jsfiddle.net/0nbsdtac/

**Comment by joyously** [comment]
Or use `text-shadow`. Or an `invert` filter.

**Comment by 00travelgirl00** [comment]
From a design perspectiv I would prefer the example of @anevins12. 

Text-shadow is in my opinion not an elegant design solution. And with invert filter, you can't control die color of text to make sure, that it fits to the Corporate Identity with the website.

**Comment by anevins12** [comment]
That works too. I didn't realise there was a blue filter on top of the image already. 

The 0.55 opacity figure was from my own research; which always produces a 4.7:1 contrast ratio against the hero image and the text. What you're suggesting with the filters is the equivalent of this, but with blue instead of black.

**Comment by kjellr** [comment]
> The 0.55 opacity figure was from my own research; which produces a 4.7:1 contrast ratio of text on top of the whitest hero image. What you're suggesting with the filters is the equivalent of this, but with blue instead of black.

Sounds great. Thanks @anevins12!
